.github/workflows/build.yml

```diff
@@ -102,6 +102,7 @@ jobs:
               - 'packages/replay/**'
               - 'packages/replay-canvas/**'
               - 'packages/feedback/**'
+              - 'packages/wasm/**'
```

Comment:
just noticed that our PW integration tests didn't run for the WASM package, so I added it to the `browser` category for our CI job selection